lua/cheatsheet/telescope.lua

```diff
@@ -80,9 +81,37 @@ M.pick_cheat = function(opts)
             attach_mappings = function(prompt_bufnr, map)
                 actions.select_default:replace(
                     function()
-                        -- local selection = action_state.get_selected_entry()
-                        -- close telescope on Enter
+                        local selection = actions_state.get_selected_entry()
+                        local cheat = selection.value.cheatcode
+                        local description = selection.value.description
+
                         actions.close(prompt_bufnr)
+
+                        if string.len(cheat) > 1 then
+                            -- check for valid command
+
+                            cheat = cheat:match("^%s*(.-)%s*$") -- strip spaces
+                            local regex = "^:[%w ]+"
+                            local cheat_sanitized = cheat:match(regex)
+                            if cheat_sanitized ~= nil then
+                                -- execute command, previous match should already
+                                -- sanitize input and next should stop at next
+                                -- non normal character like [<>,]
+                                -- Regex might need an update in the future
+                                vim.api.nvim_feedkeys(
+                                    vim.api.nvim_replace_termcodes(
+                                        cheat_sanitized, true, false, true),
+                                    "t", true)
```

Comment:
`"n"` mode would be better here than `"t"` - there may be `cmap`s in the user's config and some people (including myself :P) swap `:` with `;` since typing commands is more common. Using `"n"` will prevent remaps from being triggered.